src/sql/workbench/contrib/editData/browser/editDataEditor.ts

```diff
@@ -671,8 +690,17 @@ export class EditDataEditor extends BaseEditor {
 	public toggleQueryPane(): void {
 		this.editDataInput.queryPaneEnabled = !this.queryPaneEnabled();
 		if (this.queryPaneEnabled()) {
+			if (this._resultsEditorContainer) {
+				this._resultsEditorContainer.style.borderTopColor = this.getColor(PANEL_BORDER);
```

Comment:
I'd get rid of these and then just make sure to call `this.updateStyles()` after the resultsEditorContainer is created (which should be overridden then set the color there instead)